inst/traduire/fr-translation.json

```diff
@@ -178,5 +178,53 @@
     "OPTIONS_CALIBRATE_METHOD_LOGISTIC_LABEL": "Logistique",
     "OPTIONS_CALIBRATE_METHOD_PROPORTIONAL_LABEL": "Proportionnelle",
     "OPTIONS_CALIBRATE_METHOD_LABEL": "Méthode d'étalonnage",
-    "LANG": "fr"
+    "LANG": "fr",
+    "PLHIV_MAP_PLOT_TITLE": "Personnes vivant avec le VIH  (15+)",
```

Comment:
Which of these translations are coming from UNAIDS and which are from an online translator?

Reply:
They have all been checked by UNAIDs